packages/tkg-clusterclass-vsphere/bundle/config/upstream/base.yaml

```diff
@@ -2100,3 +2103,30 @@ spec:
             {{- range .additionalFQDN }}
             - {{ . }}
             {{- end }}
+  - name: systemWideProxy
+    enabledIf: '{{ .proxy.systemWide }}'
+    definitions:
+    - selector:
+        apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+        kind: KubeadmControlPlaneTemplate
```

Comment:
Also need another selector to patch worker nodes. Please refer to
https://github.com/vmware-tanzu/tanzu-framework/pull/4344/files#diff-2a5ffbc41b38cc6bd32db8f4b3302451be9de63a451b7c20fa2b246d4aff64ffR2136-R2159